src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -85,16 +98,17 @@ class TizenControls extends Controls {
         // Pause Display
         if (!this.pauseDisplay) {
             const pauseDisplay = createElement(PauseDisplayTemplate());
-
             const title = new Title(model);
             title.setup(pauseDisplay.querySelector('.jw-pause-display-container'));
             this.div.appendChild(pauseDisplay);
             this.pauseDisplay = pauseDisplay;
         }
 
         // Display Buttons - Buffering
-        if (!this.displayContainer) {
+        if (!this.displayContainer || !this.div.querySelector('.jw-display')) {
```

Comment:
Can you explain why this is necessary? 

this.displayContainer` is set below and never reassigned in this module. Is it set or cleared externally?

Why do we need to check for `('.jw-display')` is there another module that can add a different display?

Reply:
Not sure how this wasn't an issue before (maybe it was overlooked or happening intermittently), but there is a call to `super.enable` above which is calling enable in controls.js. The display container gets set there, however it might not exist in controls div for tizen since a new one is created here.

Reply:
Well ,what you should do is move those above `super.enable` so that you set `this.displayContainer` here before os controls.js does.

Reply:
Are you just hiding the other display with CSS? That's not good. This is where you can prevent it from being created, by defining it first.

Reply:
No. The display container from OS does not exist because it gets overwritten and is never added to the tizen-controls div. No need to hide it with CSS.

Reply:
The div gets overwritten when I call it later. I can add additional checks in OS like checking if the div, controlbar, and settings-menu already exist, so they do not get overwritten/duplicated.